evmcc/evmcc.go

```diff
@@ -91,16 +90,16 @@ func (evmcc *EvmChaincode) Invoke(stub shim.ChaincodeStubInterface) pb.Response
 
 	var gas uint64 = 10000
 	state := statemanager.NewStateManager(stub)
-	evmCache := evm.NewState(state)
+	evmCache := evm.NewState(state, func(height uint64) []byte {
+		return []byte("BlockyMcHash")
```

Comment:
gonna need more info on this.

Reply:
No, actually this is optimal :)

We added support for the BLOCKHASH opcode in the EVM (https://github.com/hyperledger/burrow/pull/967).

You should pass a block hash (or state hash of some kind) at the requested height. You only need to maintain a lookback history of 256, see: https://github.com/hyperledger/burrow/blob/develop/execution/evm/vm.go#L648 

Not a huge deal to make this a stub for now, I would have thought.